### PR TITLE
Feature/p 108 thank you styling

### PR DESCRIPTION
**Pull request by milutxi**
check this style

### CODE REVIEW

File: src/app/components/ThankYou/index.jsx [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
   return (
-    <>
+    <div className={Styles.thanks}>
       <div>
         <Image
           src={checkActive}
           alt="Icon of check being active"
-          weight={64}
-          height={51}
+          weight={70}
```
Comment (ceridwenroberts) [inline src/app/components/ThankYou/index.jsx]: Should this be width?

**Review by ceridwenroberts — CHANGES_REQUESTED** [review]
Remove border on button

File: src/app/components/ThankYou/index.jsx (hunk 1 above)
Comment (milutxi) [reply src/app/components/ThankYou/index.jsx]: yes, vc took it wrong. fix immediately

**Review by jennynguyenoberg — APPROVED** [review]

File: src/app/components/ThankYou/index.jsx (hunk 1 above)
Comment (jennynguyenoberg) [reply src/app/components/ThankYou/index.jsx]: Changed ✅

**Review by jennynguyenoberg — APPROVED** [review]

**Review by jennynguyenoberg — APPROVED** [review]

**Review by jennynguyenoberg — APPROVED** [review]

**Review by jennynguyenoberg — APPROVED** [review]

**Review by JTKsson — APPROVED** [review]

**Pull request merged**